### PR TITLE
Add experimental ordering to integration tests

### PR DESCRIPTION
**Pull request by mkam039**

### CODE REVIEW

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -48,3 +48,7 @@ jobs:
           fal run --profiles-dir .
           echo "*** Checking fal output files ***"
           python check_outputs.py
+          echo "*** Starting fal run with experimental ordering ***"
+          fal run --experimental-ordering --profiles-dir .
```
Comment (chamini2) [inline .github/workflows/integration_tests.yml:52]: Wouldn't we want to change the code then so that it changes the ordering a bit?

Comment (turbo1912) [reply .github/workflows/integration_tests.yml:52]: yea we can keep this to test it doenst error. but we can add more integration tests for the ordering

**Comment by chamini2** [comment]
What are we doing with this PR?

**Comment by mkam039** [comment]
> What are we doing with this PR?

I think we can merge as is so we can make sure the experimental ordering isn't completely broken. And then we think of tests that target ordering in particular.

**Comment by chamini2** [comment]
OK, let's do that. I'll let you merge since it's yours

**Pull request merged**